scripts/content-types/load-content-types.ts

```diff
@@ -0,0 +1,252 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const fs = require('fs');
+const readline = require('readline');
+
+const JWP_HOST = process.env.JWP_HOST || 'https://api.jwplayer.com';
+
+interface SchemaFile {
+  fields: {
+    [key: string]: Field;
+  };
+  sections: {
+    [key: string]: Section<Field | string>;
+  };
+  schemas: Schema<Field | string, Section<Field | string> | string>[];
+}
+
+// TODO: Use yup to validate schema
+interface Field {
+  label: string;
+  param: string;
+  required?: boolean;
+  details: {
+    field_type: 'input' | 'toggle';
```

Comment:
Should we also include other types here?

Reply:
I removed all of the properties besides the ones we are using.